src/main/java/org/folio/repository/CustomFieldsRepositorympl.java

```diff
@@ -0,0 +1,76 @@
+package org.folio.repository;
+
+import static org.folio.repository.CustomFieldsConstants.COUNT_CUSTOM_FIELDS_BY_REF_ID;
+import static org.folio.repository.CustomFieldsConstants.CUSTOM_FIELDS_TABLE;
+
+import java.util.Optional;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.ext.sql.ResultSet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.folio.rest.jaxrs.model.CustomField;
+import org.folio.rest.persist.PostgresClient;
+
+@Component
+public class CustomFieldsRepositorympl implements CustomFieldsRepository{
```

Comment:
Minor typo in class name - missing I in Impl

Reply:
thanks, will fix